### ISSUE TITLE
SelectableLazyColumn has no styling

### ISSUE THREAD

**Issue by rock3r** [JetBrains/jewel#158]
While `SelectableLazyColumn` is a "pure" component, it has no built-in styling system for the rows. This forces every single implementer to do:

```kotlin
val color = when {
    isSelected && isActive -> retrieveColorOrUnspecified("List.selectionBackground")
    isSelected && !isActive -> retrieveColorOrUnspecified("List.selectionInactiveBackground")
    else -> Transparent
}

Row(
    modifier = Modifier.height(IntUiTheme.globalMetrics.rowHeight)
        .background(color)
        .padding(horizontal = 4.dp),
    verticalAlignment = Alignment.CenterVertically,
    horizontalArrangement = Arrangement.spacedBy(4.dp),
) {
    // ...
```

While `IntUiTheme.globalMetrics.rowHeight` has been added (at least on a working branch), it's pretty annoying having to keep track of the state and background colors, setting the height by hand, etc etc

**Comment by hamen** [comment]
This relates to #627

**Comment by hamen** [comment]
Closed with #653
